<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CGeneralSettings</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CGeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="loadMediaInfo">
     <property name="title">
      <string>Load Media Information?</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="backgroundLoadMediaInfo">
        <property name="text">
         <string>Load Media Information in the Background?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="enableLogging">
     <property name="title">
      <string>Log Output to File?</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="logDir"/>
      </item>
      <item>
       <widget class="QToolButton" name="logDirBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/open.png</normaloff>:/SABUtilsResources/open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loadMediaInfo</tabstop>
  <tabstop>backgroundLoadMediaInfo</tabstop>
  <tabstop>enableLogging</tabstop>
  <tabstop>logDir</tabstop>
  <tabstop>logDirBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
